src/ngraph/runtime/cpu/op/quantize.cpp

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+* Copyright 2017-2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "ngraph/runtime/cpu/op/quantize.hpp"
+
+ngraph::op::Quantize::Quantize(std::shared_ptr<Node> input,
+                               std::shared_ptr<Node> min,
+                               std::shared_ptr<Node> max,
+                               const element::Type& type)
+    : RequiresTensorViewArgs("Quantize", {input, min, max})
+    , m_element_type(type)
+{
+    add_output(type, input->get_shape());
+    add_output(element::f32, Shape{1});
```

Comment:
Shouldn't this be element_type of input?

Reply:
Its not that generic at the moment. It'l only do float --> some xyz datatype.